file/helper_other.go

```diff
@@ -37,8 +37,10 @@ func SafeFileRotate(path, tempfile string) error {
 	// filesystems, so to update the parents directory metadata to actually
 	// contain the new file being rotated in.
 	f, err := os.Open(parent)
+
+	// nolint: nilerr // ignore error
 	if err != nil {
-		return nil // ignore error
+		return nil
```

Comment:
I'm not an expert here, but this looks like the kind of error that would be nice to at least log and move on. I can imagine it generating a very hard to debug edge-case bug.

Reply:
It's kind of out of scope of this PR, I think :)